server/stream.go

```diff
@@ -100,6 +100,10 @@ type StreamConfig struct {
 	// TODO(nat): Can/should we name these better?
 	ConsumerLimits StreamConsumerLimits `json:"consumer_limits"`
 
+	// AllowMsgTTL allows header initiated per-message TTLs. If disabled,
+	// then the `NATS-TTL` header will be ignored.
```

Comment:
Update comment if we reject these vs ignore.

Reply:
At the moment the comment is correct, as of right now the header is passed through and ignored. We probably want to figure out the sourcing/mirroring behaviour at the same time as changing this.

Reply:
I saw the thread between you and R.I. and Byron. From an app perspective if I set a header and get no feedback it is not honored that is not good IMO.

Yes we need to fixup the sourcing and mirroring to ignore certain headers. It does so already on some but we need a better solution here.